.buildkite/scripts/lifecycle/pre_command.sh

```diff
@@ -12,17 +12,10 @@ else
   source .buildkite/scripts/common/setup_node.sh
   source .buildkite/scripts/common/setup_buildkite_deps.sh
 
-  echo '--- Agent Debug/SSH Info'
+  echo '--- Agent Debug Info'
```

Comment:
We can remove this all entirely, the metrics are going to a different cluster now.  Links are available in the `Agent information from gobld` annotation.